misc/config_tools/data/cfl-k700-i7/hybrid_rt.xml

```diff
@@ -106,7 +106,7 @@
     </communication_vuart>
     <pci_devs>
       <pci_dev>00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection (7) I219-LM (rev 10)</pci_dev>
-      <pci_dev>09:00.0 Non-Volatile memory controller: Silicon Motion, Inc. Device 2263 (rev 03)</pci_dev>
+      <pci_dev>01:00.0 Non-Volatile memory controller: Silicon Motion, Inc. Device 2263 (rev 03)</pci_dev>
```

Comment:
Is this a new board? (Curious as to what caused the PCI bus to be different)

Reply:
board XML is obtained from the same board